Update website with quick start and notebook setup (#97)
IBM/the-mesh-for-data: website/content/docs/setup/quickstart/index.md

weight: 10
---

This guide lets you quickly evaluate {{< name >}} using the builtin sample connectors and data access module. Instaling {{< name >}} control plane includes: 
- Installing Hashicorp Vault and connector as the credentials manager 
- Installing ODPi Egeria lab and connectors as the data catalog
- Installing Open Policy Agent (OPA) and connector as the policy manager.
- Arrow-Flight data access module for reading Parquet and CSV datasets

> For a custom setup follow the [Installation instructions](/docs/setup/install)

Reviewer: These are not the all considered control plane. The quick evaluation is for setting a complete environment including third parties. I would suggest to rephrase.

```suggestion
This guide lets you quickly evaluate {{< name >}}. For a custom setup follow the [Installation instructions](/docs/setup/install).

By the end of this guide you will have the following installed to your Kubernetes cluster:
- The control plane of {{< name >}}
- Hashicorp Vault and connector as the credentials manager 
- ODPi Egeria lab and connectors as the data catalog
- Open Policy Agent (OPA) and connector as the policy manager.
- Arrow-Flight data access module for reading Parquet and CSV datasets

> The following sample was verified with the following software versions:
> - Kubernetes: kind v0.10.0, OpenShift 4.3
> You will also need to have
> - git
> 
> The following steps assume you have the following available:
> - Kubernetes cluster
> - Kubectl

Reviewer: Add header and reformat. For example:

```suggestion
# Before you begin

Ensure that you have the following:
- `kubectl` with access to a Kubernetes cluster (this guide was tested with kind v0.10.0 and OpenShift 4.3)
- `git`
- `make`
```

> 

Reviewer: Add header

1.  Obtain a local copy of {{< name >}} repository
    ```bash
    git clone https://github.com/IBM/the-mesh-for-data.git
    ```
1.  Change to the root directory of the repository
    ```bash
    cd the-mesh-for-data
    ```

> 1. Set Mesh for Data namespace

Reviewer: ```suggestion
1. Set the current namespace to `m4d-system`
```

    ```bash
    kubectl config set-context --current --namespace=m4d-system
    ```

> 1. Install {{< name >}} control plane including the built-in sample connectors and software:
>     - Credentials manager: Hashicorp Vault (aka Vault)
>     - Data catalog: ODPi Egeria (aka Egeria)
>     - Policy manager: Open Policy Agent (aka OPA)

Reviewer: Too much test, we already listed those. Consider replacing with:
```suggestion
1. Run the quick install script
```

>     Install on OpenShift:
>     ```bash
>     WITHOUT_OPENSHIFT=false ./hack/install.sh
>     ```
>     **Or**
>     
>     Install on Kind:
>     ```bash
>     ./hack/install.sh
>     ```

Reviewer: tested on kind and openshift but we shouldn't say "install on kind". Consider:
```suggestion
    ```bash
    ./hack/install.sh
    ```
   OpenShift users need to run `WITHOUT_OPENSHIFT=false ./hack/install.sh` instead.
```

> 1. Install the Arrow-Flight module

Reviewer: consider changing to
```suggestion
1. Enable the use of the [arrow flight module](https://{{< github_base >}}/the-mesh-for-data-flight-module)
```

    ```
    kubectl apply -f https://raw.githubusercontent.com/IBM/the-mesh-for-data-flight-module/master/module.yaml
    ```

> You can now start using {{< name >}}. For samples please see:

Reviewer: add next steps header?

---

This guide lets you quickly evaluate {{< name >}}. For a custom setup follow the [Installation instructions](/docs/setup/install).

## Before you begin
Ensure that you have the following:
- `kubectl` with access to a Kubernetes cluster (this guide was tested with kind v0.10.0 and OpenShift 4.3)
- `git`
- `make`

## About this guide
By the end of this guide you will have the following installed to your Kubernetes cluster:
- The control plane of {{< name >}}
- Hashicorp Vault and connector as the credentials manager 
- ODPi Egeria lab and connectors as the data catalog
- Open Policy Agent (OPA) and connector as the policy manager.
- Arrow-Flight data access module for reading Parquet and CSV datasets

## Prepare for installing {{< name >}}

1.  Obtain a local copy of {{< name >}} repository
    ```bash
    git clone https://{{< github_base >}}/{{< github_repo >}}.git
    ```
1.  Change to the root directory of the repository
    ```bash
    cd the-mesh-for-data
    ```

> 1. Set the current namespace to `m4d-system`

Reviewer: Should this move to the next section?

Author: sure.

---

This guide lets you quickly evaluate {{< name >}}. For a custom setup follow the [Installation instructions](/docs/setup/install).

## Before you begin
Ensure that you have the following:
- `kubectl` with access to a Kubernetes cluster (this guide was tested with kind v0.10.0 and OpenShift 4.3)
- `git`
- `make`

> ## About this sample

Reviewer: About this guide? Not a sample

By the end of this guide you will have the following installed to your Kubernetes cluster:
- The control plane of {{< name >}}
- Hashicorp Vault and connector as the credentials manager 
- ODPi Egeria lab and connectors as the data catalog
- Open Policy Agent (OPA) and connector as the policy manager.
- Arrow-Flight data access module for reading Parquet and CSV datasets

## Prepare for installing {{< name >}}

1.  Obtain a local copy of {{< name >}} repository
    ```bash

>     git clone https://github.com/IBM/the-mesh-for-data.git

Reviewer: Use shortcodes

Author: what do you mean?

Reviewer: https://ibm.github.io/the-mesh-for-data/contribute/documentation/write/syntax/#add-links-to-content-on-github
